ocs_ci/ocs/machine.py

```diff
@@ -3,7 +3,7 @@
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs.resources.ocs import OCS
 from ocs_ci.ocs import constants, defaults
-from tests.helpers import TimeoutSampler
+from ocs_ci.utility.utils import TimeoutSampler
```

Comment:
This is to prevent circular import error - conftest imports tests.helpers, helpers now import ocs.node, node under the hood import machine, machine would like to import helpers.